libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1188,11 +1191,11 @@ void AP_OSD_Screen::draw_message(uint8_t x, uint8_t y)
     }
 }
 
-void AP_OSD_Screen::draw_speed_vector(uint8_t x, uint8_t y,Vector2f v, int32_t yaw)
+void AP_OSD_Screen::draw_speed_vector(uint8_t x, uint8_t y,Vector2f v, int32_t yaw, bool force_arrow)
```

Comment:
change this to be a function that takes a body frame vector, and add a comment explaining what it does

Reply:
Moved to a function that takes a angle and magnitude